### DISCUSSION TITLE
Theme Ionic >=v.2 - Administração de Negócios

### DISCUSSION THREAD

**Discussion by carloshenriqueribeiro** [frontendbr/forum#776] [Perguntas e Respostas]
Galera, boa noite! 

Estou procurando um tema de administração de negócios em Ionic, alguém conhece algum open e free ?

Encontrei muitos templates para e-commerce mas para administração do negócio tá bem difícil.

O objetivo é ter um ponto de start não, necessariamente,  encontrar algo completo (se existir, melhor ainda..rs)

Exemplo: simples cadastro de cliente com aprovação;

_exemplo sem ser ionic:_
http://akveo.com/blur-admin/#/dashboard

Desde já agradeço!

**Comment by mumumilk** [comment]
Em ionic propriamente eu não achei.

Mas tem o site creativetim que possui umas dashboards legais.
Essa que vou passar é free e em angular, talvez tu consiga aproveitar, já que o ionic é baseado nesse framework.

https://www.creative-tim.com/product/paper-dashboard-angular
